archinstall/default_profiles/desktops/plasma.py

```diff
@@ -16,7 +16,11 @@ def packages(self) -> list[str]:
 			"konsole",
 			"kate",
 			"dolphin",
+			"dolphin-plugins",
+			"flatpak-kcm",
+			"kcmutils",
 			"ark",
+			"kf6",
```

Comment:
[kf6](https://archlinux.org/groups/x86_64/kf6/) is a pretty massive group of applications. All of which feels optional.